### DISCUSSION TITLE
WYSIWYG Editor for writing translations

### DISCUSSION THREAD

**Discussion by jarlatsimplesite** [WeblateOrg/weblate#4822] [Ideas]
One single thing missing for us in weblate is a simple WYSIWYG editor.
Not a full blown one with table, image etc support. Just a simple one where we can insert a link, mark bold. Set color etc.

Seeing the age of weblate and the usage I guess there is a good reason for this not being implemented.
We just cannot rely on our translators writing tags correctly so having a wysiwyg would make this perfect

**Comment by nijel** [comment]
It would be certainly useful for some use cases, but most people do not translate HTML and do not need such features.

The editor should be however switched to CodeMirror soon (currently, it is scheduled for 4.4) and that could bring at least some comfort for editing HTML (see https://github.com/WeblateOrg/weblate/issues/4218 on related topic).

**Reply by nijel** [reply]
It has been implemented and Weblate 4.4 will provide syntax highlighting for HTML (and others).

**Reply by robertvy** [reply]
Hi @nijel, we use Weblate but need more robust HTML translation. Are you aware of any suitable open-source solution we could run besides Weblate? Thank you!

**Reply by nijel** [reply]
Unfortunately, I don't have any expertise in this.

**Reply by jasonbrown1965** [reply]
Love to look into weblate as a newsroom option in a region with 20+ languages, but lack of WYSIWYG kinda kills it in terms of onboarding, or any kind of slack/teams etc integration ?
